src/QMCDrivers/Crowd.h

```diff
@@ -44,6 +44,9 @@ class Crowd
    */
   Crowd(EstimatorManagerNew& emb,
         const DriverWalkerResourceCollection& driverwalker_res,
+	const ParticleSet& pset,
+        const TrialWaveFunction& twf,
+	const QMCHamiltonian& hamiltonian_temp,
```

Comment:
I feel very bad about adding this "gold" element here. Could you leave a comment about why they are added here. Probably need to get rid of them eventually.